### DISCUSSION TITLE
Information for historical calendars

### DISCUSSION THREAD

**Discussion by JohnRDOrazio** [Liturgical-Calendar/LiturgicalCalendarAPI#149] [General]
1) Calendar of Filocalo, [Chronograph of 354](https://en.wikipedia.org/wiki/Chronograph_of_354), contains the **Depositio Martyrum** as well as the **Depositio Episcoporum**. It can well be considered the first Liturgical Calendar in history.
   * source: [Furius Dionysius Filocalus, Kalendarium Anno CCCLIV Conscriptum](https://github.com/Liturgical-Calendar/LiturgicalCalendarAPI/blob/ec56c6f3439ba5c7f1085887a95f9a509f2eba4a/literature/0354-0354,_Furius_Dionysius_Filocalus,_Kalendarium_Anno_CCCLIV_Conscriptum,_MLT.pdf)

1) The **Verona Sacramentary** is the oldest known liturgical book. It is not a sacramentary in the strict sense, but rather a private collection of libelli missarum (missal booklets) containing only the prayers for certain Masses. It is sometimes called "Leonine" because it has been attributed to Pope Leo I († 461). The three [quires](https://en.wikipedia.org/wiki/Units_of_paper_quantity#Quire) containing the period 1 January – 14 April are lost. Thus, there is no information in the earliest Roman liturgical book concerning the Easter Vigil.
   * source: ???

1) The **Gelasian Sacramentary** is the second oldest western [liturgical book](https://en.wikipedia.org/wiki/Liturgical_book) that has survived (Paris, **750 AD**).
   * source: ???

1) The first move to unify the Roman liturgy took place under Charlemagne:

   > Among several distinct rites current in the West before the 8th century, the two most influential were the [Roman rite](https://en.wikipedia.org/wiki/Roman_rite) used in Italy south of Lombardy and the [Gallican](https://en.wikipedia.org/wiki/Gallican_rite) in use in most of the rest of Western Europe, save Iberia and the British Isles. By 700 the influence of the Roman sacramentary had modified Gallican usage. This mixture of rites represented in the Gelasian Sacramentary was superseded when Charlemagne asked Pope Hadrian to provide an authentic Roman sacramentary for use throughout the empire. In **785-86**, the pope sent the emperor the **Sacramentarium Hadrianum**, a version of the Gregorian Sacramentary for papal use, which was adapted for the Carolingian empire.

   * **Sacramentarium Hadrianum** (785): https://www.latinitas.it/grh/

1) The first printed **Missale Romanum** (Roman Missal), containing the **Ordo Missalis secundum consuetudinem Curiae Romanae** (Order of the Missal in accordance with the custom of the [Roman Curia](https://en.wikipedia.org/wiki/Roman_Curia)), was produced in **Milan** in **1474**.

   Annotations in the hand of **Cardinal** [**Gugliemo Sirleto**](https://en.wikipedia.org/wiki/Gugliemo_Sirleto) in a copy of the 1494 Venetian edition[[4]](https://en.wikipedia.org/wiki/Roman_Missal#cite_note-4) show that it was used for drawing up the 1570 official edition of **Pope Pius V**. In substance, this 1494 text is identical with that of the 1474 Milanese edition.
   * source: https://archive.org/details/missaleromanumme01cath

1) Implementing the decision of the Council of Trent, **Pope Pius V** promulgated, in the Apostolic Constitution [Quo primum](https://en.wikipedia.org/wiki/Quo_primum) of **14 July 1570**, an edition of the Roman Missal that was to be in obligatory use throughout the Latin Church except where there was another liturgical rite that could be proven to have been in use for at least two centuries.
   * source: [SS Pius V, Missale Romanum](https://github.com/Liturgical-Calendar/LiturgicalCalendarAPI/blob/6c1185c1cfac59bef59601b4c0112b3968f75290/literature/1570-07-19,_SS_Pius_V,_Missale_Romanum,_LT.pdf)

1) Some corrections to Pope Pius V's text proved necessary, and **Pope Clement VIII** replaced it with a new typical edition of the Roman Missal on 7 July **1604**. (In this context, the word "typical" means that the text is the one to which all other printings must conform.)
   * source: http://books.google.it/books?id=u7xFAAAAcAAJ&printsec=frontcover&dq=Missale&hl=it&ei=qk5ATu6vEoup8APij73-Ag&sa=X&oi=book_result&ct=result&resnum=2&ved=0CDIQ6AEwAQ#v=onepage&q&f=false

1) A further revised typical edition was promulgated by **Pope Urban VIII** on 2 September **1634**.
   * source: https://www.digitale-sammlungen.de/de/view/bsb10869696

**Comment by JohnRDOrazio** [comment]
Coming to modern times:

* Missale Romanum ex decreto Sacrosancti Concilii Tridentini restitutum 1957: http://www.clerus.org/bibliaclerusonline/it/hc.htm
* Missale Romanum ex decreto Sacrosancti Concilii Tridentini restitutum 1962: http://www.clerus.org/bibliaclerusonline/it/zr.htm

* Missale Romanum 1970 (Editio typica)
* Missale Romanum 1971 (Reimpressio emendata)
* Missale Romanum 1975 (Editio typica secunda)
* Missale Romanum 2002 (Editio typica tertia): http://www.clerus.org/bibliaclerusonline/es/1d.htm
* Missale Romanum 2008 (Editio typica tertia emendata)

**Comment by ptrsmk** [comment]
I am hoping to make some contributions to assist in making the API usable for the 1962 Missal.  This would be not just to make requests from those years, but also to make it possible to generate a calendar for the current year according to the 1962 Missal.  I don't want to do anything in vain or in a manner that is less than helpful or inefficient.  So, would you mind letting me know, first off, if this would be welcome, and secondly, whether you might provide some info on how you would like contributions and discussions?

**Reply by JohnRDOrazio** [reply]
I would be interested in extending the API back in time for historical reasons, to be able to compare the calendar between years. In order to implement pre-1970 Missal data, I would need to see a comparison of all the calendar changes from the 1962 Missal to the 1970 Editio Typica.
Requesting 1962 Missal data in the current year is not possible, the API is not designed to accomplish this. That would require a different API.

**Comment by JohnRDOrazio** [comment]
There is a good [overview on the USCCB website](https://www.usccb.org/prayer-and-worship/the-mass/general-instruction-of-the-roman-missal/roman-missal):

1604 – Pope Clement VIII
1634 – Pope Urban VIII
1884 – Pope Leo XIII
1920 – Pope Benedict XV
1962 – Pope Saint John XXIII
1970 – Pope Saint Paul VI
1975 – Pope Sanint Paul VI
2002 – Pope Saint John Paul II

In addition, there were a number of other minor revisions to the text, published as "reprints" which incorporated minor changes.  The most recent of these were in 1957 after Pope Pius XII's revisions to the rites of Holy Week and the Sacred Triduum in 1955, and in 2008, when Pope Benedict XVI incorporated a number of additional prayers, included those for recently canonized saints as well as for the celebration of an extended Vigil for Pentecost.
